airbyte-webapp/src/components/ArrayOfObjectsEditor/components/EditorRow.tsx

```diff
@@ -2,9 +2,9 @@ import React from "react";
 import { useIntl } from "react-intl";
 
 import { Button } from "components";
+import { Tooltip } from "components/base/Tooltip";
```

Comment:
I like the idea: `base` is really generic, to the point that it doesn't convey much at all except "some of this might get overridden, maybe". `ui` would make it clearer to me what other components I could expect to find in that directory, which already seems like a win.